webapp/src/features/adminEditUser/AdminEditUser.tsx

```diff
@@ -62,11 +58,11 @@ const AdminEditUser = (props: AdminEditUserProps) => {
     },
   })
 
-  const { userData } = props
+  const { email } = useParams<{ email: string }>()
 
   useEffect(() => {
-    dispatch(getUserData(userData.email))
-  }, [userData, dispatch])
+    dispatch(getUserData(email))
```

Comment:
We'll want a check for invalid edit attempts (non-existent users)

Reply:
Added a check!